src/target/riscv/riscv.c

```diff
@@ -2867,8 +2867,16 @@ static int riscv_address_translate(struct target *target,
 
 		uint8_t buffer[8];
 		assert(info->pte_shift <= 3);
-		int retval = r->read_memory(target, pte_address,
-				4, (1 << info->pte_shift) / 4, buffer, 4);
+		riscv_mem_access_info_t read_info = {
```

Comment:
Calling this `info` is IMO not bad but I was thinking if I can come up with a more fitting name.

Maybe `args` (memory access arguments)?

I'll leave it up to you whether to use `args` or keep `info`.

```suggestion
		riscv_mem_access_args_t read_args = {
```

Reply:
Thanks. Addressed